### PR TITLE
Fix #74: Fixing multiple tags for same address issue

### LINKED ISSUE #74: Reported labels show the same address for the same source
**Describe the bug**
For some wallets, when you go to reported labels you'll find the same label, for the same source and address.

**To Reproduce**
1. Go to https://toshi.cibr.qcri.org/wallet/5222e061-8180-4c5b-b03d-81f35d08bf7a
2. Click on Reported Labels

**Expected behavior**
If these are exactly the same labels, they should not be displayed as duplicates. The only way they're different is if the "notes" are different in the database, and this is the case for Dizzy labels where an address is labeled the same from different URLs (we save the URLs as notes).

I cannot think of another case now, but if we have labels that are all the same except notes, then we should group them and display a special message there (the info icon) next to source, saying "Found xx tags for the same label and address from different darkweb URLs" or something similar to that.

**Screenshots**

Screen Shot 2022-06-05 at 8 55 20 AM

### PR DESCRIPTION
**Pull request by sajithaliyanage**
- Closes https://github.com/cibr-qcri/toshi/issues/74

### CODE REVIEW

File: web/src/components/DataTableRaw/DataTableRaw.js [hunk 1]
```diff
@@ -50,19 +52,37 @@ export const DataTableRaw = (props) => {
 
   // Renderers
   const renderCell = (column, row) => {
+    let cellText;
+    if (column.id === 'source' && row.count > 1) {
+      cellText = (
+        <Typography className={classes.infoItem}>
+          {row[column.id]}
+          <Tooltip
+            title={`Found ${row.count} reported tags from different darkweb URLs`}
```
Comment (boshmaf) [inline web/src/components/DataTableRaw/DataTableRaw.js:61]: Found xx reports from different darkweb URLs

File: api/middleware/wallet/labels.js [hunk 2]
```diff
@@ -32,11 +33,12 @@ const walletLabels = asyncHandler(async (request, response, next) => {
 
   const addresses = walletLabelsRes.rows.map((row) => {
     return {
-      id: row.id,
+      id: util.format('%s-%s-%s', row.source, row.label, row.address),
```
Comment (boshmaf) [inline api/middleware/wallet/labels.js:36]: Just use the row index as an ID for react. We don't use this ID anywhere.

File: api/middleware/wallet/labels.js [hunk 3]
```diff
@@ -2,6 +2,7 @@ const asyncHandler = require('../../middleware/async');
 const ErrorResponse = require('../../utils/errorResponse');
 const gp = require('../../services/gp');
 const wallet = require('../../utils/wallet');
+const util = require('util');
```
Comment (boshmaf) [inline api/middleware/wallet/labels.js:5]: Please remove as it's not needed anymore

**Review by boshmaf — CHANGES_REQUESTED** [review]

File: web/src/components/DataTableRaw/DataTableRaw.js (hunk 1 above)
Comment (sajithaliyanage) [reply web/src/components/DataTableRaw/DataTableRaw.js]: Fixed

File: api/middleware/wallet/labels.js (hunk 2 above)
Comment (sajithaliyanage) [reply api/middleware/wallet/labels.js]: Fixed

File: api/middleware/wallet/labels.js (hunk 3 above)
Comment (sajithaliyanage) [reply api/middleware/wallet/labels.js]: Fixed

**Review by boshmaf — APPROVED** [review]

**Pull request merged**